templates/vrrp_track_process.erb

```diff
@@ -0,0 +1,15 @@
+vrrp_track_process <%= @name %> {
```

Comment:
since this is a new template, could you provide it as epp? In our docs, we agreed on using epp instead of erb for new templates:
https://voxpupuli.org/docs/#reviewing-a-module-pr
> Is a new template added? The preferred language is [epp](https://puppet.com/docs/puppet/latest/lang_template_epp.html), not erb